game/menu/gamedisplay.gd

```diff
@@ -51,5 +51,5 @@ func _handle_error(err):
 
 
 func _on_loadbutton_pressed():
-	if GameManager.load_game(game_file)!=OK:
+	if GameManager.load_game(game_file) != OK:
```

Comment:
Even though both changes are minimal, this mixes two different things, which can (theoretically) kind of create a dependency/conflict nightmare. So just as a suggestion, focus on single (or at least related) issues per PR in the f uture.

Reply:
I disagree in this particular change, because CI is broken right now on main.

I mean creating two PR would've fixed that too, but due to the current merge criteria, this is not fast enough.

Reply:
Also both errors are introduced by my PRs, which makes me responsible.

Reply:
Yes, but I also think fixups like this shouldn't even have to go through any kind of voting as long as they're straightfoward and 100% within pre-defined rulesets (i.e. fixing stuff that slipped through).